packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -17,7 +17,7 @@ export default createRule({
     type: 'suggestion',
     docs: {
       description:
-        'Require switch-case statements to be exhaustive with union type',
+        'Require switch-case statements to be exhaustive with union type or enum',
```

Comment:
(same existing typo-ish)

```suggestion
        'Require switch-case statements to be exhaustive with union types or enums',
```